apps/delivery-options/src/views/MyParcelDeliveryOptions/Pickup/PickupLocationOpeningHours/PickupLocationOpeningHours.vue

```diff
@@ -5,69 +5,60 @@
       v-text="translate(OPENING_HOURS)" />
 
     <div
-      v-for="hours in filteredOpeningHours"
+      v-if="openingHours.length === 0"
+      class="mp-content-between"
+      v-text="translate(CLOSED)"></div>
```

Comment:
ESLint should make you use an inline closing tag here. Please re-enable ESLint and lint all the files you've changed. 

IntellIJ IDEs sometimes give an error when trying to lint .vue files, to "fix" it for a bit, restart the ESLint service.